app/models/mixins/ui_service_mixin.rb

```diff
@@ -1,21 +1,24 @@
 module UiServiceMixin
+  # rubocop:disable Metrics/LineLength
```

Comment:
Nope, it's on purpose. I wanted to talk about it with you actually. If the file is breaking any rubocop rule from any reason (mostly code readability), I guess it's the best thing to do to break the rule too when adding a new thing to it, right? On the other hand there is a lot of 'false positives' in the report.

What is wrong with

```ruby
# rubocop:enable Metrics/LineLength
 very long lines, blah, blah
 very long lines, blah, blah
...
# rubocop:disable Metrics/LineLength
```

`routes.rb` is another example or `default_menu.rb`